VideoLocker/res/layout/row_course_list.xml

```diff
@@ -214,22 +175,42 @@
                 android:layout_height="wrap_content"
                 android:layout_gravity="center_vertical"
                 android:ellipsize="end"
+                android:singleLine="true"
                 android:textColor="@color/org_code_txt"
-                android:textSize="12sp"
-                android:singleLine="true"/>
+                android:textSize="12sp" />
         </LinearLayout>
     </RelativeLayout>
 
-    <!-- put a frame over this cell, so that corners look rounded -->
+    <org.edx.mobile.view.custom.ETextView
+        android:id="@+id/course_certified_banner"
+        style="@style/bold_white_text"
+        android:layout_width="180dp"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_marginTop="17dp"
+        android:background="@color/course_banner_blue"
+        android:gravity="center"
+        android:lineSpacingMultiplier="0.8"
+        android:paddingBottom="3dp"
+        android:paddingEnd="40dp"
+        android:paddingLeft="40dp"
+        android:paddingRight="40dp"
+        android:paddingStart="40dp"
```

Comment:
Just noting here that despite the issue with using relative and absolute padding together in RTL, if left and right paddings are the same then it will work find if you just skip the relative definitions. This is just a clarification upon our previous conversation, nothing needs to be fixed.